jsons/Policies.json

```diff
@@ -331,13 +331,14 @@
                     "Free [Great Prophet] appears",
                     "[+1 Food, +3 Culture] from every [Holy site]",
                     "Provides a [Garden] in your first [4] cities for free",
-                    "May buy [Great Engineer] units for [1000] [Faith] [in capital] at an increasing price ([500]) <starting from the [Industrial era]>",
-                    "May buy [Great General] units for [1000] [Faith] [in capital] at an increasing price ([500]) <starting from the [Industrial era]>",
-                    "May buy [Great Artist] units for [1000] [Faith] [in capital] at an increasing price ([500]) <starting from the [Industrial era]>",
-                    "May buy [Great Writer] units for [1000] [Faith] [in capital] at an increasing price ([500]) <starting from the [Industrial era]>",
-                    "May buy [Great Musician] units for [1000] [Faith] [in capital] at an increasing price ([500]) <starting from the [Industrial era]>",
-                    "May buy [Great Admiral] units for [1000] [Faith] [in capital] at an increasing price ([500]) <starting from the [Industrial era]>",
-                    "May buy [Great Merchant] units for [1000] [Faith] [in capital] at an increasing price ([500]) <starting from the [Industrial era]>"
+                    "May buy [Great Engineer] units for [1000] [Faith] [in capital] at an increasing price ([1500]) <starting from the [Industrial era]> <before adopting [Tradition Complete]>",
+                    "May buy [Great General] units for [1000] [Faith] [in capital] at an increasing price ([1500]) <starting from the [Industrial era]> <before adopting [Honor Complete]>",
+                    "May buy [Great Artist] units for [1000] [Faith] [in capital] at an increasing price ([1500]) <starting from the [Industrial era]> <before adopting [Aesthetics Complete]>",
+                    "May buy [Great Writer] units for [1000] [Faith] [in capital] at an increasing price ([1500]) <starting from the [Industrial era]> <before adopting [Aesthetics Complete]>",
+                    "May buy [Great Musician] units for [1000] [Faith] [in capital] at an increasing price ([1500]) <starting from the [Industrial era]> <before adopting [Aesthetics Complete]>",
+                    "May buy [Great Admiral] units for [1000] [Faith] [in capital] at an increasing price ([1500]) <starting from the [Industrial era]> <before adopting [Exploration Complete]>",
+                    "May buy [Great Merchant] units for [1000] [Faith] [in capital] at an increasing price ([1500]) <starting from the [Industrial era]> <before adopting [Commerce Complete]>"
+                    // quess here need more elegant solution to make a limit for every great person = 1
```

Comment:
Wait... Really? Are the costs actually different before and after adopting that policy branch's policies? That sounds incorrect

Reply:
Yes, I discovered it by accident. If you make a different price in the policies, then the one that was received later is applied

Reply:
I feel like the question here is "is that intentional?" For me, I would have 2 questions before I could ok this
1. Does this happen in both directions? E.g., if you finish honor first, then Piety, which price do we use? If it's Honor's, then I guess this is fine. If it's Piety's, this sounds like a bug
2. Is this an intended interaction? Because this very much reads like something that is very wrong, even if it was intentional

Reply:
hm, when I was debugging for the first time, I caught a bug. But now i can't reproduce it.. everything works as you describe, so <before adopting [Aesthetics Complete]> isn't need here. 

And as for the fact that I just increased the price (500->1500) increase instead of limiting every great person(1), is that okay? I think it needs to be cut off somehow, even if there is no perfect solution right now.

Reply:
Tbh, I hadn't fact checked the increase until you mentioned it just now... Hmm... That probably should be done via a fake building, like Lekmod handles it. I'm not if there's a better solution there besides a fake building

Reply:
I added [Great [People] Altar] only available <after adopting [Piety Complete]>.
What do you think?

Reply:
#213

Reply:
I see, canceled my solution. e0bf25cd7fce3383291c4c6fce439a6a1a60dec9